rust/datafusion/src/physical_plan/functions.rs

```diff
@@ -71,6 +71,8 @@ pub enum Signature {
     Exact(Vec<DataType>),
     /// fixed number of arguments of arbitrary types
     Any(usize),
+    /// One of a list of signatures
+    OneOf(Vec<Signature>),
```

Comment:
FYI @seddonm1  I am not sure how this affects your string functions / other postgres function plans

Reply:
Yes, I missed this but all good. This is actually better :D